src/OpenSage.Game/OpenSage.Game.csproj

```diff
@@ -39,7 +39,7 @@
   <ItemGroup>
     <ShaderFiles Include="Assets\Shaders\*.vert;Assets\Shaders\*.frag" />
     <UpToDateCheckInput Include="@(ShaderFiles)" />
-    
+
```

Comment:
Those newlines probably can be removed again

Reply:
Just checked the project file, it always had new lines, should we remove all extra new lines, or you meant restore those removed spaces so they don't appear in the git diff?

Reply:
Ah you just removed spaces. Didn't look correctly then :) Just keep it as is.

Reply:
So I'll just keep the "remove space" change?

Reply:
Yes